base/shell.jl

```diff
@@ -81,7 +81,7 @@ function shell_parse(raw::AbstractString, interp::Bool)
                     if done(s,k)
                         error("unterminated double quote")
                     end
-                    if s[k] == '"' || s[k] == '$'
+                    if s[k] == '"' || s[k] == '$' || s[k] == '\\'
```

Comment:
Especially since that fix will need to be incorporated into @nolta's port of `shell_parse` to Scheme.